gobblin-data-management/src/main/java/gobblin/data/management/retention/profile/ConfigurableGlobDatasetFinder.java

```diff
@@ -64,6 +65,15 @@
 
   private static final Map<String, String> DEPRECATIONS = ImmutableMap.of(DATASET_FINDER_PATTERN_KEY,
       DATASET_PATTERN_KEY, DATASET_FINDER_BLACKLIST_KEY, DATASET_BLACKLIST_KEY);
+
+  public ConfigurableGlobDatasetFinder(FileSystem fs, Properties jobProps, Config config) throws IOException {
+    this.fs = fs;
+    this.blacklist = Optional.absent();
```

Comment:
Yes. With the config management integration, blacklists are not at the datasetFinder level. A blacklist is decided by the dataset itself.